src/Symfony/Component/PropertyAccess/Tests/PropertyAccessorCollectionTest.php

```diff
@@ -23,7 +23,9 @@ public function __construct($axes = null)
         $this->axes = $axes;
     }
 
-    // In the test, use a name that StringUtil can't uniquely singularify
+    /**
+     * In the test, use a name that StringUtil can't uniquely singularify.
```

Comment:
this is not a doc comment of the method. It is a comment on the implementation details